HISTORY.md

```diff
@@ -1,349 +1,353 @@
-v3.2.4
```

Comment:
remove formatting diff.

Reply:
I need to turn off my prettier, it is reformatting, will change